src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -40,6 +40,8 @@
                 ],
     'Imper': [0, 0, 0, 0, 0, 0, 0, 0, 0, 0,         # Impervious surface area percentage
               0.15, 0.52, 0.87, 0.15, 0.52, 0.87],  # only defined for urban land use types
+    'C': [0.03, 0.42, 0.002, 0.01, 0.1, 0.03, 0.04, 0.001, 0.01, 0.08,  # C Factor Defaults
```

Comment:
Thanks, I think I used a word document that BME sent. This is more correct, though.